src/domains/ecommerce-common/03_storage.tf

```
@@ -271,7 +271,7 @@ resource "azurerm_monitor_scheduled_query_rules_alert" "ecommerce_transient_enqu
     custom_webhook_payload = "{}"
   }
   data_source_id = module.ecommerce_storage_transient.id
-  description    = format("%s enqueue rate  > 10 per ${each.value.time_window} minutes",replace("${each.value.queue_key}","-"," "))
+  description    = format("Enqueuing rate for queue %s > ${each.value.threshold} during last ${each.value.time_window} minutes",replace("${each.value.queue_key}","-"," "))
```

```suggestion
  description    = format("Enqueuing rate for queue %s > ${each.value.threshold} during last ${each.value.time_window} minutes", replace("${each.value.queue_key}","-"," "))
```

(if not overridden by terraform formatter)